### ISSUE TITLE
Ensure spell check is enabled for smaller data entry boxes 

### ISSUE THREAD

**Issue by magdalenadrafiova** [DMPRoadmap/roadmap#2169]
From our accessibility testing the following was mentioned: 
The smaller text boxes e.g. what project are you planning did not have spellcheck enabled but the larger free text boxes e.g. Project Abstract did.

**Comment by sjDCC** [comment]
I'm not 100% sure how to test this one. On one plan I have I can see 'NERC' underlined in red in a project details abstract. This may be coming from browser rather than an accessibility feature in tool.

Any advice @johnpinto1

**Comment by magdalenadrafiova** [comment]
I am sorry I do not know how to test this one - I am adding this to our meeting on Tuesday morning so Sam/Ray can show us

**Comment by johnpinto1** [comment]
@sjDCC @magdalenadrafiova  spellcheck: "true" is a HTML 5 attribute that has been added to all tags. So the browser if it supports HTML 5 will highlight spelling errors for your locale.

**Comment by magdalenadrafiova** [comment]
Now when I go into create a new project and I start filling the details there is a spell check enabled for the front page - for the title of the new project. Seems fine closing this.

**Comment by mariapraetzellis** [comment]
Reopening b/c we're not seeing the feature in tinymce boxes.

**Comment by xsrust** [comment]
Having a look at the types of spellchecking that TinyMCE supports, I think I'd still rather go with the browser-based package rather than us running a spellchecking service.  See: https://www.tiny.cloud/docs/general-configuration-guide/spell-checking/ for more details on the options given to us by TinyMCE.

**Comment by mariapraetzellis** [comment]
I'm not clear if this was enabled @xsrust? I don't see the spell check feature in data entry boxes. 

I have heard from several users that they would like to see a spell check feature, so unless it's a real drag on the application I think it is worth enabling. My Chrome browser was not picking up spelling errors when filling out a test DMP in the app. 

![screenshot-dmponline-dev dcc ac uk-2020 02 03-15_09_20](https://user-images.githubusercontent.com/27511575/73698881-31ef6500-4697-11ea-96b2-09e20ae86fcc.png)

**Comment by magdalenadrafiova** [comment]
that has not been deployed just yet

**Comment by magdalenadrafiova** [comment]
seems to be working @mariapraetzellis  you want to try too? 
![screenshot-dmponline-dev dcc ac uk-2020 02 04-17_04_21](https://user-images.githubusercontent.com/47178106/73768138-6edb5a80-4770-11ea-9f43-e357aebcc2e3.png)

**Comment by sjDCC** [comment]
Looks good for me too. Closing out

**Comment by mariapraetzellis** [comment]
Looks good to me as well.

**Comment by dsisu** [comment]
Sorry to have to re-open this but the spell checker still doesn't work for all data entry boxes. For example, notification title box (it works for the body box).

**Comment by johnpinto1** [comment]
Will look at this now.

**Comment by pherterich** [comment]
@dsisu this works for me now, are you okay to confirm and close out or update with details of where you still encounter issues please?
